imports/plugins/core/orders/client/components/OrderRefunds.js

```diff
@@ -0,0 +1,303 @@
+import React, { Fragment, useEffect, useRef, useState } from "react";
+import PropTypes from "prop-types";
+import { Mutation } from "react-apollo";
+import { Form } from "reacto-form";
+import withStyles from "@material-ui/core/styles/withStyles";
+import Card from "@material-ui/core/Card";
+import CardContent from "@material-ui/core/CardContent";
+import CardHeader from "@material-ui/core/CardHeader";
+import FormControl from "@material-ui/core/FormControl";
+import InputLabel from "@material-ui/core/InputLabel";
+import Grid from "@material-ui/core/Grid";
+import MenuItem from "@material-ui/core/MenuItem";
+import OutlinedInput from "@material-ui/core/OutlinedInput";
+import Select from "@material-ui/core/Select";
+import Typography from "@material-ui/core/Typography";
+import ErrorsBlock from "@reactioncommerce/components/ErrorsBlock/v1";
+import Field from "@reactioncommerce/components/Field/v1";
+import TextInput from "@reactioncommerce/components/TextInput/v1";
+import { i18next, Reaction } from "/client/api";
+import formatMoney from "/imports/utils/formatMoney";
+import ConfirmButton from "/imports/client/ui/components/ConfirmButton";
+import createRefundMutation from "../graphql/mutations/createRefund";
+import OrderPreviousRefunds from "./OrderPreviousRefunds";
+
+const styles = (theme) => ({
+  dividerSpacing: {
+    marginBottom: theme.spacing(4),
+    marginTop: theme.spacing(4)
+  },
+  fontWeightSemiBold: {
+    fontWeight: theme.typography.fontWeightSemiBold
+  },
+  formControl: {
+    minWidth: "250px",
+    width: "250px"
+  },
+  selectEmpty: {
+    marginTop: theme.spacing(2)
+  }
+});
+
+/**
+ * @name OrderRefunds
+ * @param {Object} props Component props
+ * @returns {React.Component} returns a React component
+ */
+function OrderRefunds(props) {
+  const hasPermission = Reaction.hasPermission(["reaction-orders", "order/fulfillment"], Reaction.getUserId(), Reaction.getShopId());
```

Comment:
I would remove this and the two places where `hasPermission` are checked. We should let the server decide. But if the server does respond with a permission error, then we should show a helpful error message, e.g., "You don't have permission to do this"

Reply:
How are we doing server-to-client alerts / error messages with GraphQL? I scanned the docs and didn't come across anything.

Reply:
Added using `InlineError`. Created a ticket to further explore better error messaging: https://github.com/reactioncommerce/reaction/issues/5426